README.md

```diff
@@ -207,13 +207,6 @@ All rights reserved.
      <https://tools.ietf.org/html/rfc8446>
 
 <!-- Logos and Badges -->
-<!--
-  Note: The security token for the appveyor badge (the random number in
-  the URL below) was obtained for the mspncp/openssl project.
-  It needs to be replaced by the correct token by some core member
-  (@levitte, @mattcaswell?). It can be obtained for project members at
-  https://ci.appveyor.com/project/openssl/openssl/settings/badges.
--->
```

Comment:
@openssl/omc can somebody of you please do me a favour and send my the correct access token for the appveyor badge? It can be obtained only by a GitHub project owner of the openssl/openssl project.

Reply:
How does one get the access token?

Reply:
I would expect that you just have to follow the [link](https://ci.appveyor.com/project/openssl/openssl/settings/badges) mentioned in the comment and login with your GitHub account if you are not logged in already. If you have the correct permissions, you should see something like the following:

![image](https://user-images.githubusercontent.com/11467854/87932478-dd487680-ca8b-11ea-88ff-925051408da6.png)

Reply:
I just tried it: If the login screen reappears, you don't have the necessary permissions.